### PR TITLE
New os Alcoma

### PR DESCRIPTION
**Pull request by erotel**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
Thanks @erotel. All looks good, just a question on the os name. It's usually not a good idea to base it on the vendor name. Looking at the firmware for this device it doesn't appear to be called anything but to safeguard the vendor building a new product line should we use `alcoma-al`?

**Comment by erotel** [comment]
this device does not support ssh or telnet, so I do not know the system name.
The firmware is the same for both the AL series and the MP series.
the older ALxxMP / ME series (MPS, ME, ME32, MP90) does not support snmp queries wirelessly.
Can I renaming it to "Alcoma_almp"?

**Comment by laf** [comment]
@erotel seems fine with me, `alcoma-almp` though to be consistent.

File: LibreNMS/OS/AlcomaAlmp.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Alcoma.php
```
Comment (laf) [inline LibreNMS/OS/AlcomaAlmp.php]: For completeness may as well change this

**Pull request merged**